### PR TITLE
Copy stake flow- create the old `TokenStaking` web3 contract instance

### PR DESCRIPTION
**Pull request by r-czajkowski**
Depends on: #1989 

For `local` and `Ropsten` network, we want to use the "stub" version of `TokenStaking` from `@keep-network/keep-core` artifact([`OldTokenStaking`](https://github.com/keep-network/keep-core/blob/master/solidity/contracts/stubs/OldTokenStaking.sol)). Against `mainnet`, we want to use `TokenStaking` artifact from `1.1.2` version at [`0x6D1140a8c8e6Fac242652F0a5A8171b898c67600`](https://etherscan.io/address/0x6D1140a8c8e6Fac242652F0a5A8171b898c67600#code) address.

### CODE REVIEW

File: solidity/dashboard/src/contracts.js [hunk 1]
```diff
@@ -235,3 +240,17 @@ export function getBondedECDSAKeepFactoryAddress() {
...
+    process.env.NODE_ENV === "development" ||
+    getFirstNetworkIdFromArtifact() === "3"
+  ) {
+    // For local and ropsten network we want the mock old `TokenStaking` contract.
+    return (
+      await import("@keep-network/keep-core/artifacts/OldTokenStaking.json")
+    ).default
+  }
+
+  return (await import("./old-contracts-artifacts/TokenStaking.json")).default
```
Comment (nkuba) [inline solidity/dashboard/src/contracts.js:255]: What does this `default` do? I'm just curious.

File: solidity/dashboard/src/contracts.js [hunk 2]
```diff
@@ -235,3 +240,17 @@ export function getBondedECDSAKeepFactoryAddress() {
 export function getTBTCSystemAddress() {
   return getContractAddress(TBTCSystem)
 }
+
+const getOldTokenStakingArtifact = async () => {
+  if (
+    process.env.NODE_ENV === "development" ||
+    getFirstNetworkIdFromArtifact() === "3"
```
Comment (nkuba) [inline solidity/dashboard/src/contracts.js:247]: Let's add just a simple clarification comment.
```suggestion
    getFirstNetworkIdFromArtifact() === "3" // ropsten
```

File: solidity/dashboard/src/contracts.js (hunk 1 above)
Comment (r-czajkowski) [reply solidity/dashboard/src/contracts.js:255]: `await import(*)` returns `module object` that contains all exports. pls [see](https://javascript.info/modules-dynamic-imports#the-import-expression).

File: solidity/dashboard/src/contracts.js [hunk 3]
```diff
@@ -235,3 +240,17 @@ export function getBondedECDSAKeepFactoryAddress() {
...
+  ) {
+    // For local and ropsten network we want the mock old `TokenStaking` contract.
```
[the remark this answers is not in the document]
Comment (nkuba) [reply solidity/dashboard/src/contracts.js:249]: We can't as we need to use matching `TokenGrant` and `KeepToken` contracts and our CI process doesn't support that.

File: solidity/dashboard/src/contracts.js (hunk 2 above)
Comment (r-czajkowski) [reply solidity/dashboard/src/contracts.js]: pls see 5b9453d

**Review by nkuba — DISMISSED** [review]

File: solidity/dashboard/src/contracts.js [hunk 4]
```diff
@@ -235,3 +240,18 @@ export function getBondedECDSAKeepFactoryAddress() {
...
+const getOldTokenStakingArtifact = async () => {
+  if (getFirstNetworkIdFromArtifact() === "1") {
+    // Mainnet network ID.
+    // Against mainnet, we want to use TokenStaking artifact
+    // from 1.1.2 version at `0x6D1140a8c8e6Fac242652F0a5A8171b898c67600` address.
+    return (await import("./old-contracts-artifacts/TokenStaking.json")).default
+  }
+
+  // For local, Ropsten and keep-dev network we want to use
+  // the mocked old `TokenStaking` contract from `@keep-network/keep-core` pacakge.
```
Comment (nkuba) [inline solidity/dashboard/src/contracts.js:253]:
```suggestion
  // the mocked old `TokenStaking` contract from `@keep-network/keep-core` package.
```

**Review by nkuba — APPROVED** [review]

**Pull request merged**